batcher/aligned/src/main.rs

```diff
@@ -107,20 +109,51 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
     #[arg(name = "Private key", long = "private_key")]
     private_key: Option<String>,
+    #[arg(name = "Max Fee", long = "max_fee")]
+    max_fee: Option<String>, // String because U256 expects hex
     #[arg(
-        name = "Max Fee",
-        long = "max_fee",
-        default_value = "1300000000000000" // 13_000 gas per proof * 100 gwei gas price (upper bound)
+        name = "Price Estimate",
+        long = "price_estimate",
+        default_value = "default",
+        help = "`price_estimate` allows for a user to use the aligned_sdk `estimate_fee` function to compute there `max_fee`.\n Usage: `default,`, `instant`, \"custom <NUM_PROOFS_IN_BATCH>\"",
```

Comment:
I would explain this a bit more, here is a suggestion:

```suggestion
        help = "`price_estimate` allows for a user to use the aligned_sdk `estimate_fee` function to compute the `max_fee` based on the number of proofs in a batch and the current gas price from the `rpc_node`.\n Usage: `default,`, `instant`, \"custom <NUM_PROOFS_IN_BATCH>\"",
```